agendas/2022/2022-07-14.md

```diff
@@ -105,6 +105,7 @@ Arda Tanrikulu          | @ardatan               | The Guild              | Ista
 Saihajpreet Singh       | @saihaj                | The Guild              | Ottawa, Canada
 Yaacov Rydzinski.       | @yaacovcr              | Individual.            | Neve Daniel, IL
 Tim Suchanek            | @timsuchanek           | Stellate               | Berlin, Germany
+Matteo Collina          | @mcollina              |                        | Italy
```

Comment:
Hi Matteo, preferably add the company you're associated with but failing that, please enter "Individual":

```suggestion
Matteo Collina          | @mcollina              | Individual             | Italy
```